src/openpgp/ocfb.iced

```diff
@@ -279,9 +279,9 @@ exports.Decryptor = Decryptor
 
 {rng} = require 'crypto'
 test = () ->
-  plaintext = new Buffer("a man a plan a canal panama. and you know the rest")
+  plaintext = Buffer.from("a man a plan a canal panama. and you know the rest", 'utf8')
```

Comment:
Looks like default encoding is `utf8` for both `new Buffer` and `Buffer.from`, but good call on passing it explicitly.